### PR TITLE
Create Binary.py

### PR DESCRIPTION
**Pull request by SDakshit**
Binary Search in Python.

### CODE REVIEW

File: Python/Search/Binary.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+def binary(arr, low, high, n):
```
Comment (amanmehara) [inline Python/Search/Binary.py:1]: name it `binary_search`.

File: Python/Search/Binary.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+def binary(arr, low, high, n): 
+	if high >= low: 
+
```
Comment (amanmehara) [inline Python/Search/Binary.py:3]: Indent the code correctly.

**Review by amanmehara — CHANGES_REQUESTED** [review]
Please have a look at https://github.com/amanmehara/programming/blob/master/CONTRIBUTING.md and make changes accordingly.
I'll approve and merge afterwards! 😃

**Comment by SDakshit** [comment]
Sir, I made the changes. Please have a look. PR #101

**Pull request closed without merging**